README.md

```diff
@@ -18,203 +18,203 @@
 ![feature-sliced-banner](/website/static/img/banner.jpg)
 
 <!-- ⚡ Add primary information & features about your repository -->
-Методология для проектирования *frontend проектов*, нацеленная [**на разделение приложения согласно бизнес-логике и областям ответственности.**][refs-splitting]
+A methodology for designing *frontend projects*, aimed [**at dividing the application according to business logic and areas of responsibility.**][refs-splitting]
 
-- Обеспечивает [**понятность, контролируемость и адаптивность**][refs-arch-req] архитектуры
-- Основана на [**проверенных временем**][refs-motivation-why] практиках проектирования и концепциях
+- Provides [**explicitness, controllability and adaptability**][refs-arch-req] architecture
+- Based on [**experience-checked**][refs-motivation-why] design practices and concepts
     > `SOLID`, `GRASP`, `DDD`, `Separation of Concerns`, `Vertical Slices`, `Public API`, `Isolation`
-- Предлагает разделять проект согласно [**бизнес-юнитам**][ext-ubiq-lang]
+- Suggests dividing the project according to [**business units**][ext-ubiq-lang]
 
-> **Примечание:** Методология не привязана к конкретному стеку и применима *к любым фронтенд-проектам* в целом.
+> **Note:** The methodology is not tied to a specific stack and is applicable *to any frontend projects* in general.
 >
-> Но текущая версия приводит примеры и проработана на базе связки `JavaScript` + `React`
+> But the current version gives examples and is developed on the basis of a bunch of `JavaScript` + `React`
 
 ## Motivation
 
-Обычно, подходы построения архитектуры фронтенда от проекта к проекту [переизобретаются с нуля][refs-motivation], пополняя тем самым ["проектные знания"][refs-knowledge]
+Usually, the approaches to building the frontend architecture from project to project are [re-invented from scratch][refs-motivation], thereby adding ["project knowledge"][refs-knowledge]
 
-> Несмотря на то, что специфика фронтенд-проектов отличается не так значительно
+> Despite the fact that the specifics of frontend projects do not differ so much
 
-При этом, неверно принятые решения зачастую приводят [к проблемам масштабируемости проекта и команды][refs-arch-problems].
+At the same time, incorrectly made decisions often lead [to problems of scalability of the project and the team][refs-arch-problems].
 
-И поэтому, вместо того, чтобы придумывать и документировать это каждый раз, хочется **обобщить опыт и сформировать рабочую, проверенную и задокументированную методологию** для проектирования архитектуры фронтенда.
+And therefore, instead of inventing and documenting it every time, I want to **summarize the experience and form a working, proven and documented methodology** for designing the frontend architecture.
```

Comment:
Мб как-то так?
```suggestion
And therefore, instead of inventing and documenting it every time, it is better to **summarize the experience and form a working, proven and documented methodology** for designing the frontend architecture.
```